guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -25,38 +25,44 @@ include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[le
 endif::[]
 
 // Upgrading Satellite Server overview
-include::topics/upgrading_overview.adoc[leveloffset=+1]
+include::topics/con_upgrading-overview.adoc[leveloffset=+1]
```

Comment:
How about moving the new files into `common/modules`?

Reply:
Hey @Lennonka I am planning to do that in a separate PR.